docs/data-collection/scanning.md

```diff
@@ -39,6 +39,39 @@
     - [ ] check the ECG and RB signals, and fix unanticipated problems (e.g., the respiration belt needs to be fastened tighter);
     - [ ] the *Amphetamine* app is running and keeping the computer unlocked while *AcqKnoledge* is working.
 
+- [ ] Check the quality of physiological signals
+
+    Please find in the following image examples of good and bad RB and CO2 signals to support and illustrate the criteria to consider to assess the quality of the physiological signals.
+
+    ??? thanks "Thank you to Cesar Caballero-Gaudes, Stefano Moia, Kristina Zvolankek, Elenor Morgenroth for putting together the slide below [1]."
+    ![qa_physio](../assets/images/qa_physio.png)
+
+    - [ ] Check the respiration belt signal
+        - [ ] The RB signal should not plateau (nor on the peaks, nor on the troughs). This means the RB is maxing out and needs to be loosen.
+        - [ ] The envelope of your signal should remain approximately constant. If the peaks drop or the trough rise, it might be a sign that the RB is too loose. Go back in the scanning room and tighten the RB.
```

Comment:
```suggestion
        - [ ] The envelope of your signal SHOULD remain approximately constant.
            If peaks drop or troughs rise, it might be a sign that the RB is too loose.
            Go back into the scanning room and tighten the RB.
```